### ISSUE TITLE
Unknown Distribution on RHEL 8.1

### ISSUE THREAD

**Issue by ImNtReal** [r-darwish/topgrade#365]
**What did you expect to happen?**
...
Topgrade to run

**What actually happened?**
...
Error detecting current distribution: Unknown Linux Distribution

**Additional details:**
- Which operating system or Linux distribution are you using?
```
(ansible) [ansible@vminansible01 ~]$ cat /etc/os-release
NAME="Red Hat Enterprise Linux"
VERSION="8.1 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.1"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.1 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.1:GA"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.1
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.1"
```
- How did you install Topgrade?
```
cargo install topgrade
```
<details><pre>
(ansible) [ansible@vminansible01 ~]$ topgrade -V
Topgrade 4.2.0
(ansible) [ansible@vminansible01 ~]$ topgrade -v
 2020-03-10T14:10:35.432Z DEBUG topgrade::config > Configuration at /home/ansible/.config/topgrade.toml
 2020-03-10T14:10:35.432Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: None, commands: None, git_repos: None, predefined_git_repos: None, disable: None, remote_topgrades: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: Some(true), notify_each_step: None, accept_all_windows_updates: None, only: None }
 2020-03-10T14:10:35.432Z DEBUG topgrade::utils  > Cannot find "notify-send"
 2020-03-10T14:10:35.432Z DEBUG topgrade         > Version: 4.2.0
 2020-03-10T14:10:35.432Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2020-03-10T14:10:35.432Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2020-03-10T14:10:35.432Z DEBUG topgrade         > Binary path: Ok("/home/ansible/.cargo/bin/topgrade")
 2020-03-10T14:10:35.432Z DEBUG topgrade         > Self Update: false
 2020-03-10T14:10:35.432Z DEBUG topgrade::utils  > Detected "/bin/git" as "git"
 2020-03-10T14:10:35.432Z DEBUG topgrade::utils  > Detected "/bin/sudo" as "sudo"
 2020-03-10T14:10:35.432Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2020-03-10T14:10:35.432Z DEBUG topgrade::utils  > Cannot find "powershell"
Error detecting current distribution: Unknown Linux Distribution
 2020-03-10T14:10:35.433Z DEBUG topgrade::runner > Step "etc-update"
 2020-03-10T14:10:35.433Z DEBUG topgrade::utils  > Cannot find "etc-update"
 2020-03-10T14:10:35.433Z DEBUG topgrade::runner > Step "brew"
 2020-03-10T14:10:35.433Z DEBUG topgrade::utils  > Cannot find "brew"
 2020-03-10T14:10:35.433Z DEBUG topgrade::runner > Step "nix"
 2020-03-10T14:10:35.433Z DEBUG topgrade::utils  > Cannot find "nix"
 2020-03-10T14:10:35.433Z DEBUG topgrade::runner > Step "home-manager"
 2020-03-10T14:10:35.433Z DEBUG topgrade::utils  > Cannot find "home-manager"
 2020-03-10T14:10:35.433Z DEBUG topgrade::runner > Step "asdf"
 2020-03-10T14:10:35.433Z DEBUG topgrade::utils  > Cannot find "asdf"
 2020-03-10T14:10:35.433Z DEBUG topgrade::steps::git > /home/ansible/.doom.d does not exists
 2020-03-10T14:10:35.433Z DEBUG topgrade::steps::git > /home/ansible/.vim does not exists
 2020-03-10T14:10:35.433Z DEBUG topgrade::steps::git > /home/ansible/.config/nvim does not exists
 2020-03-10T14:10:35.433Z DEBUG topgrade::steps::git > /home/ansible/.zshrc is a file. Checking /home/ansible
 2020-03-10T14:10:35.433Z DEBUG topgrade::steps::git > Checking if /home/ansible is a git repository
 2020-03-10T14:10:35.435Z TRACE topgrade::executor   > Output of "/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.tmux does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.config/fish does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.config/openbox does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.config/bspwm does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.config/i3 does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::steps::git > /home/ansible/.config/sway does not exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "Git repositories"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "zr"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.zr" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "antibody"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "antibody"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "antigen"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.zshrc" exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/antigen.zsh" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "zplug"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.zshrc" exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/zplug" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "zinit"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.zshrc" exists
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.zinit" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Detected "/bin/zsh" as "zsh"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.oh-my-zsh" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "fisher"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "fish"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "tmux"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Path "/home/ansible/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "TLDR"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "tldr"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "rustup"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "rustup"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "cargo"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "cargo-install-update"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "Flutter"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "flutter"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "Go"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "go"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "Emacs"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "emacs"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "opam"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "opam"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "vcpkg"
 2020-03-10T14:10:35.435Z DEBUG topgrade::utils      > Cannot find "vcpkg"
 2020-03-10T14:10:35.435Z DEBUG topgrade::runner     > Step "pipx"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "pipx"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "stack"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "stack"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "tlmgr"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "tlmgr"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "myrepos"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "mr"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "pearl"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "pearl"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "jetpak"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Cannot find "jetpack"
 2020-03-10T14:10:35.436Z DEBUG topgrade::runner     > Step "vim"
 2020-03-10T14:10:35.436Z DEBUG topgrade::utils      > Detected "/bin/vim" as "vim"
 2020-03-10T14:10:35.439Z TRACE topgrade::executor   > Output of "/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Jul 23 2019 13:28:34)\nIncluded patches: 1-1763\nModified by <bugzilla@redhat.com>\nCompiled by <bugzilla@redhat.com>\nHuge version without GUI.  Features included (+) or not (-):\n+acl               +farsi             +mouse_sgr         -tag_any_white\n+arabic            +file_in_path      -mouse_sysmouse    -tcl\n+autocmd           +find_in_path      +mouse_urxvt       +termguicolors\n-autoservername    +float             +mouse_xterm       +terminal\n-balloon_eval      +folding           +multi_byte        +terminfo\n+balloon_eval_term -footer            +multi_lang        +termresponse\n-browse            +fork()            -mzscheme          +textobjects\n++builtin_terms    +gettext           +netbeans_intg     +timers\n+byte_offset       -hangul_input      +num64             +title\n+channel           +iconv             +packages          -toolbar\n+cindent           +insert_expand     +path_extra        +user_commands\n-clientserver      +job               +perl/dyn          +vertsplit\n-clipboard         +jumplist          +persistent_undo   +virtualedit\n+cmdline_compl     +keymap            +postscript        +visual\n+cmdline_hist      +lambda            +printer           +visualextra\n+cmdline_info      +langmap           +profile           +viminfo\n+comments          +libcall           +python/dyn        +vreplace\n+conceal           +linebreak         +python3/dyn       +wildignore\n+cryptv            +lispindent        +quickfix          +wildmenu\n+cscope            +listcmds          +reltime           +windows\n+cursorbind        +localmap          +rightleft         +writebackup\n+cursorshape       +lua/dyn           +ruby/dyn          -X11\n+dialog_con        +menu              +scrollbind        -xfontset\n+diff              +mksession         +signs             -xim\n+digraphs          +modify_fname      +smartindent       -xpm\n-dnd               +mouse             +startuptime       -xsmp\n-ebcdic            -mouseshape        +statusline        -xterm_clipboard\n+emacs_tags        +mouse_dec         -sun_workshop      -xterm_save\n+eval              +mouse_gpm         +syntax            \n+ex_extra          -mouse_jsbterm     +tag_binary        \n+extra_search      +mouse_netterm     +tag_old_static    \n   system vimrc file: \"/etc/vimrc\"\n     user vimrc file: \"$HOME/.vimrc\"\n 2nd user vimrc file: \"~/.vim/vimrc\"\n      user exrc file: \"$HOME/.exrc\"\n       defaults file: \"$VIMRUNTIME/defaults.vim\"\n  fall-back for $VIM: \"/etc\"\n f-b for $VIMRUNTIME: \"/usr/share/vim/vim80\"\nCompilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H     -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -I/usr/include/python3.6m -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       \nLinking: gcc   -L. -Wl,-z,relro  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -fstack-protector-strong -rdynamic -Wl,-export-dynamic -Wl,--enable-new-dtags -Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld  -Wl,-z,relro  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -L/usr/local/lib -Wl,--as-needed -o vim        -lm  -lselinux   -lncurses -lacl -lattr -lgpm -ldl   -Wl,--enable-new-dtags -Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -Wl,-z,relro -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -fstack-protector-strong -L/usr/local/lib  -L/usr/lib64/perl5/CORE -lperl -lpthread -lresolv -ldl -lm -lcrypt -lutil -lc        \n", stderr: "" }
 2020-03-10T14:10:35.439Z DEBUG topgrade::runner     > Step "Neovim"
 2020-03-10T14:10:35.439Z DEBUG topgrade::utils      > Cannot find "nvim"
 2020-03-10T14:10:35.439Z DEBUG topgrade::runner     > Step "voom"
 2020-03-10T14:10:35.439Z DEBUG topgrade::utils      > Cannot find "voom"
 2020-03-10T14:10:35.439Z DEBUG topgrade::runner     > Step "NPM"
 2020-03-10T14:10:35.439Z DEBUG topgrade::utils      > Cannot find "npm"
 2020-03-10T14:10:35.439Z DEBUG topgrade::runner     > Step "yarn"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "yarn"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "composer"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "composer"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "apm"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "apm"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "gem"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "gem"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "Flatpak"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "flatpak"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "snap"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "snap"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "pihole"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "pihole"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "fwupdmgr"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "Restarts"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Cannot find "needrestart"
 2020-03-10T14:10:35.440Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Detected "/bin/bash" as "bash"
 2020-03-10T14:10:35.440Z DEBUG topgrade::utils      > Path "/home/ansible/.sdkman/bin/sdkman-init.sh" doesn't exist
</pre></details>

**Comment by ImNtReal** [comment]
Is there a way to force it to try to run as a particular distro?

**Comment by ImNtReal** [comment]
I'm not sure if it's how you want to handle it, but this got it to at least run for me: https://github.com/r-darwish/topgrade/pull/366

**Comment by r-darwish** [comment]
Fixed in #366
